corehq/apps/users/templates/users/partials/edit_role_modal.html

```diff
@@ -274,13 +274,13 @@ <h4 class="modal-title" data-bind="text: modalTitle"></h4>
           </div>{# END data permissions #}
           
           {# BEGIN web apps permissions #}
-          <div class="form-group">
+          <div class="form-group" data-bind="visible: $root.webAppsPrivilege">
```

Comment:
I'd have done this in the template with `{% if web_apps_privilege %}` because it isn't going to vary on the same page load. But this works.